workflows/rules/variantcalling/pindel.smk

```diff
@@ -0,0 +1,105 @@
+        output:
+            pindelConfig = "{workingdir}/{stype}/pindel/{sname}_pindelConfig.txt"
+        shell:
+            "echo '{input.tumorbam}\t300\t{tumorname}'>{output.pindelConfig}"
+
+rule pindel:
+    input:
+        pindelConfig = expand("{workingdir}/{stype}/pindel/{sname}_pindelConfig.txt", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"])
+    params:
+        bed = pipeconfig["rules"]["pindel"]["bed"],
+        reference = pipeconfig["referencegenome"],
+        threads = clusterconf["pindel"]["threads"],
+        x = 2,
+        B = 60
+    singularity:
+        pipeconfig["singularities"]["pindel"]["sing"]
+    output:
+        "{workingdir}/{stype}/pindel/{sname}_BP",
+        "{workingdir}/{stype}/pindel/{sname}_CloseEndMapped",
+        "{workingdir}/{stype}/pindel/{sname}_D",
+        "{workingdir}/{stype}/pindel/{sname}_INT_final",
+        "{workingdir}/{stype}/pindel/{sname}_INV",
+        "{workingdir}/{stype}/pindel/{sname}_LI",
+        "{workingdir}/{stype}/pindel/{sname}_RP",
+        "{workingdir}/{stype}/pindel/{sname}_SI",
+        "{workingdir}/{stype}/pindel/{sname}_TD"
+    shell:
+        "echo $HOSTNAME;"
+        " (pindel -f {params.reference} -i {input.pindelConfig} -T {params.threads} -x {params.x} -B {params.B} -j {params.bed} -o {workingdir}/{wildcards.stype}/pindel/{wildcards.sname} ) "
+
+rule pindel2vcf:
+    input:
+        expand("{workingdir}/{stype}/pindel/{sname}_BP", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+        expand("{workingdir}/{stype}/pindel/{sname}_CloseEndMapped", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+        expand("{workingdir}/{stype}/pindel/{sname}_D", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+        expand("{workingdir}/{stype}/pindel/{sname}_INT_final", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+        expand("{workingdir}/{stype}/pindel/{sname}_INV", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+        expand("{workingdir}/{stype}/pindel/{sname}_LI", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+        expand("{workingdir}/{stype}/pindel/{sname}_RP", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+        expand("{workingdir}/{stype}/pindel/{sname}_SI", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+        expand("{workingdir}/{stype}/pindel/{sname}_TD", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"])
+    params:
+        threads = clusterconf["pindel"]["threads"],
+        reference = pipeconfig["referencegenome"],
+        refname = "GRCh38",
+        refdate = 000000,
+        e = 3, #e = 10,
+        mc = 10,
```

Comment:
Have these values been tested? They may be too low for wgs if they are copied from myeloid

Reply:
-e is the minimum of supporting reads required for a variant to be reported. now using 3 since one variant we were looking for in a sample only had 3 supporting reads for the alternate allele. myeloid pipeline uses 10 here…

-mc is minimum number of reads to provide a genotype. haven't thought about this one so much, 10 is default